### PR TITLE
Fix ucrt builds

### PR DESCRIPTION
**Pull request by Biswa96**

### CODE REVIEW

File: mingw-w64-grep/PKGBUILD [hunk 1]
```diff
@@ -4,25 +4,34 @@
...
-pkgver=3.4
+pkgver=3.6
 pkgrel=1
 pkgdesc="Grep searches one or more input files for lines containing a match to a specified pattern (mingw-w64)"
 arch=('any')
 mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64')
 url='https://www.gnu.org/software/grep/'
 license=('GPL3')
 depends=("${MINGW_PACKAGE_PREFIX}-pcre")
+makedepends=("${MINGW_PACKAGE_PREFIX}-winpthreads")
```
Comment (mati865) [inline mingw-w64-grep/PKGBUILD:15]: I think this should also have runtime dependency on `${MINGW_PACKAGE_PREFIX}-libwinpthread-git`.

File: mingw-w64-sed/PKGBUILD [hunk 2]
```diff
@@ -4,27 +4,32 @@
...
 pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
-pkgver=4.2.2
-pkgrel=2
+pkgver=4.4
+pkgrel=1
 pkgdesc="Sed is a stream editor (mingw-w64)"
 arch=('any')
 mingw_arch=('mingw32' 'mingw64' 'ucrt64' 'clang64')
 url='https://www.gnu.org/software/sed/'
 license=('GPL3')
```
Comment (jeremyd2019) [inline mingw-w64-sed/PKGBUILD:13]:
```suggestion
license=('GPL3')
depends=("${MINGW_PACKAGE_PREFIX}-libssp")
```

File: mingw-w64-grep/PKGBUILD (hunk 1 above)
Comment (jeremyd2019) [reply mingw-w64-grep/PKGBUILD:15]: the commit message (and my recollection of the issue) refers to needing to link winpthreads for *tests*, so I would guess the actual installed binary does not depend on libwinpthread.

Comment (Biswa96) [reply mingw-w64-grep/PKGBUILD:15]: Yup, winpthreads in grep are for tests, not for any installed binaries.

File: mingw-w64-sed/PKGBUILD (hunk 2 above)
Comment (Biswa96) [reply mingw-w64-sed/PKGBUILD:13]: Done.

**Comment by Biswa96** [comment]
CI tries to use the grep from previous build and fails. But sed and grep build fine.

File: mingw-w64-grep/PKGBUILD (hunk 1 above)
Comment (mati865) [reply mingw-w64-grep/PKGBUILD:16]: There is special array for that: https://wiki.archlinux.org/title/PKGBUILD#checkdepends

Comment (Biswa96) [reply mingw-w64-grep/PKGBUILD:16]: According to it `These dependencies are only considered when the check() function is present and is to be run by makepkg`. Those tests are built in `build()` function.

**Comment by Biswa96** [comment]
ping!

**Comment by Biswa96** [comment]
@lazka ping!

**Pull request merged**